firedrake/mg/interface.py

```diff
@@ -31,7 +31,8 @@ def prolong(coarse, fine):
         if len(Vc) != len(Vf):
             raise ValueError("Mixed spaces have different lengths")
         for in_, out in zip(coarse.split(), fine.split()):
-            prolong(in_, out)
+            myprolong, _, _ = firedrake.dmhooks.get_transfer_operators(in_.function_space().dm)
+            myprolong(in_, out)
```

Comment:
Don't hardcode the transfer operators here so that this does the right thing when we have custom ones.